### ISSUE TITLE
Application insight broke service bus library

### ISSUE THREAD

**Issue by sigmaadam** [microsoft/ApplicationInsights-node.js#775]
I'm using Azure App Insight for node.js.
I'm import and run this lib first line in my project.
All the project works well except the service bus.

When I'm trying to send message to service bus, I'm getting the following error: _a.context is not a function.

* I'm using @azure/service-bus, version 7.1.0
* My base docker-file is node:15.13.0-alpine

Stack trace:

"TypeError: _a.context is not a function\n    at Object.getSpanContext (/Users/adamve/go/src/sigmadots/iot/cloud_services/Cloud_Fog_Micro_Service/node_modules/@opentelemetry/api/build/src/context/context.js:70:76)\n    at getParent (/Users/adamve/go/src/sigmadots/iot/cloud_services/Cloud_Fog_Micro_Service/node_modules/@opentelemetry/tracing/build/src/Tracer.js:101:16)\n    at Tracer.startSpan (/Users/adamve/go/src/sigmadots/iot/cloud_services/Cloud_Fog_Micro_Service/node_modules/@opentelemetry/tracing/build/src/Tracer.js:49:31)\n    at Tracer.tracer.startSpan (/Users/adamve/go/src/sigmadots/iot/cloud_services/Cloud_Fog_Micro_Service/node_modules/diagnostic-channel-publishers/dist/src/azure-coretracing.pub.js:55:44)\n    at /Users/adamve/go/src/sigmadots/iot/cloud_services/Cloud_Fog_Micro_Service/node_modules/@azure/core-tracing/dist/index.js:727:27\n    at ServiceBusAdministrationClient.createTopic (/Users/adamve/go/src/sigmadots/iot/cloud_services/Cloud_Fog_Micro_Service/node_modules/cloud_common_utilities/node_modules/@azure/service-bus/dist/index.js:2449:42)\n    at ServiceBusAdministration.<anonymous> (/Users/adamve/go/src/sigmadots/iot/cloud_services/Cloud_Fog_Micro_Service/node_modules/cloud_common_utilities/serviceBus/serviceBusAdministration.js:69:71)\n    at Generator.next (<anonymous>)\n    at /Users/adamve/go/src/sigmadots/iot/cloud_services/Cloud_Fog_Micro_Service/node_modules/cloud_common_utilities/serviceBus/serviceBusAdministration.js:8:71\n    at new Promise (<anonymous>)\n    at __awaiter (/Users/adamve/go/src/sigmadots/iot/cloud_services/Cloud_Fog_Micro_Service/node_modules/cloud_common_utilities/serviceBus/serviceBusAdministration.js:4:12)\n    at ServiceBusAdministration.validateTopicExists (/Users/adamve/go/src/sigmadots/iot/cloud_services/Cloud_Fog_Micro_Service/node_modules/cloud_common_utilities/serviceBus/serviceBusAdministration.js:60:16)\n    at ServiceBusPublisher.<anonymous> (/Users/adamve/go/src/sigmadots/iot/cloud_services/Cloud_Fog_Micro_Service/node_modules/cloud_common_utilities/serviceBus/serviceBusPublisher.js:83:79)\n    at Generator.next (<anonymous>)\n    at /Users/adamve/go/src/sigmadots/iot/cloud_services/Cloud_Fog_Micro_Service/node_modules/cloud_common_utilities/serviceBus/serviceBusPublisher.js:8:71\n    at new Promise (<anonymous>)\n    at __awaiter (/Users/adamve/go/src/sigmadots/iot/cloud_services/Cloud_Fog_Micro_Service/node_modules/cloud_common_utilities/serviceBus/serviceBusPublisher.js:4:12)\n    at ServiceBusPublisher.initTopics (/Users/adamve/go/src/sigmadots/iot/cloud_services/Cloud_Fog_Micro_Service/node_modules/cloud_common_utilities/serviceBus/serviceBusPublisher.js:79:16)\n    at ServiceBusPublisher.<anonymous> (/Users/adamve/go/src/sigmadots/iot/cloud_services/Cloud_Fog_Micro_Service/node_modules/cloud_common_utilities/serviceBus/serviceBusPublisher.js:40:32)\n    at Generator.next (<anonymous>)\n    at /Users/adamve/go/src/sigmadots/iot/cloud_services/Cloud_Fog_Micro_Service/node_modules/cloud_common_utilities/serviceBus/serviceBusPublisher.js:8:71\n    at new Promise (<anonymous>)"

**Comment by hectorhdzg** [comment]
@sigmaadam I'm not able to reproduce this on my end, can you try with latest version of both @azure/service-bus 7.2.0 and applicationinsights 2.1.1?, are you installing @opentelemetry packages directly?

**Comment by sigmaadam** [comment]
Hi,

I found the exact flow that make the issue, the flow in only when you try send topic event within express route.

In the following versions:
@azure/service-bus 7.2.0
applicationinsights 2.1.1
express 4.17.1

example of code:
```
import * as appInsights from 'applicationinsights'
const applicationInsightKey = "APPLICATION_INSIGHT_INSTRUMENTATION_KEY"
appInsights.setup(applicationInsightKey).start()

import express from "express";
import { ServiceBusClient, ServiceBusMessage } from "@azure/service-bus";

const connectionString = "SERVICE_BUS_CONNECTION_STRING"
const serviceBusClient = new ServiceBusClient(connectionString)
const sender = serviceBusClient!.createSender('test1');

const message: ServiceBusMessage = {
    body: {'hellow':true},
    subject: 'sub'
}

const app = express();
const port = 9090; // default port to listen
// define a route handler for the default home page
app.get("/", (req, res) => {
//THIS CODE WILL THROW THE ERROR
    sender.sendMessages(message).then((res) => {
        console.log('res', res)
    }).catch(err => {
        console.error(err)
    })
} );
```

**I noticed that when I use service bus version 7.0.0 it's working.**

**Comment by hectorhdzg** [comment]
@sigmaadam problem is with OpenTelemetry API mismatch between @azure/service-bus and the one used in Application Insights, you can actually take a look at which specific version is being used by both packages navigating in node_modules folder, OpenTelemetry just published version 1.0.0 of the API and no more breaking changes are expected, we will be upgrading soon and Azure SDKs will be upgrading as well, you can see OpenTelemetry current compatibility matrix [here](https://github.com/open-telemetry/opentelemetry-js#compatibility-matrix), latest versions of the packages must not have any issue, I ran your code with no issues  but is possible some other package in your environment is adding another version of OpenTelemetry, you can also try to do a clean install to ensure node_modules is correct.

**Comment by sigmaadam** [comment]
ok, for now I solve this by downgrade the version of service-bus
